src/decls/types.js

```diff
@@ -63,3 +63,21 @@ declare type startSpanArgs = {
     tags?: any,
     startTime?: number,
 };
+
+declare type SenderResponse = {
+    err: boolean,
+    numSpans: number
+};
+
+declare class Transport {
+    append(span: Span): SenderResponse;
+    flush(): SenderResponse;
+    close(): void;
+};
+
+declare class Logger {
+    info(msg: string): void;
+    debug(msg: string): void;
+    warn(msg: string): void;
+    error(msg: string): void;
+}
```

Comment:
wait, so before you were creating whole new fils, with licenses, for each of the reporter interface, but now the logger doesn't get its own file?